client/pages/applications/[id]/score-table.tsx

```diff
@@ -7,7 +7,7 @@ import { useApplicationDetails } from 'services';
 
 const ScoreTable: NextPage = () => {
   const { query } = useRouter();
-  const { id } = query;
+  const id = query.id ? +query.id : undefined;
```

Comment:
I'm not sure here. Do you think might be better to add ? here: `query?.id`?
And remind me again, what's + for in `+query.id`?

Reply:
it should handle it gracefully, but I'll change this to `query?.id && +query.id`.

`+` is shorthand for converting to number